contracts/modules/royalty/policies/RoyaltyPolicyLAP.sol

```diff
@@ -72,9 +72,11 @@ contract RoyaltyPolicyLAP is IRoyaltyPolicyLAP, AccessManagedUpgradeable, Reentr
     }
 
     /// @notice Initializer for this implementation contract
-    /// @param governance The governance address
-    function initialize(address governance) external initializer {
-        __AccessManaged_init(governance);
+    /// @param accessManager The address of the protocol admin roles contract
+    function initialize(address accessManager) external initializer {
+        if (accessManager == address(0)) revert Errors.RoyaltyPolicyLAP__ZeroAccessManager();
```

Comment:
Do we need to do this check for all other modules?  @Ramarti ?

Reply:
Would say that if so - perhaps we can add it in a separate pr @Ramarti